class-tgm-plugin-activation.php

```diff
@@ -2023,12 +2064,31 @@ public function force_activation() {
 					} elseif ( $this->can_plugin_activate( $slug ) ) {
 						// There we go, activate the plugin.
 						activate_plugin( $plugin['file_path'] );
+						$this->append_transient( 'tgmpa_force_activated_plugins', $slug );
 					}
 				}
 			}
 		}
 
 		/**
+		 * Set/update a transient, appending the new value if the transient already exists.
+		 *
+		 * @since 2.6.0
+		 *
+		 * @param string       $transient_key Name of transient.
+		 * @param string|array $data To set or add to transient.
```

Comment:
Sorry to be a pain, but the parameter comments should line up, so the text `To set..` needs some more spaces before it.